syntax = "proto3";

package sensor;
//python -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. sensor.proto

service SensorControl {
  rpc SendCommand (CommandRequest) returns (CommandResponse);
}

message CommandRequest {
  string command = 1; // "ligar" ou "desligar"
}

message CommandResponse {
  string message = 1;
}